### PR TITLE
Feature/logger plugin

### PR DESCRIPTION
**Pull request by StyleT**

### CODE REVIEW

File: ilc/server/errorHandler/ErrorHandler.spec.js [hunk 1]
```diff
@@ -60,13 +71,24 @@ describe('error handler', () => {
             '</body>' +
             '</html>'
         );
-        chai.expect(JSON.parse(global.console.error.getCall(0).args[0])).to.deep.include({
+
+        const output = parseOutout(stdoutInspect.output);
```
Comment (nightnei) [inline ilc/server/errorHandler/ErrorHandler.spec.js:75]: "output"

File: ilc/server/pluginManager/PluginManager.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                console.warn(`Plugin installed at path "${pluginPath}" of type "${pSource.type}" was ignored as ` +
+                    `it duplicates the existing one.`);
+                continue;
+            }
+
+            console.info(`Enabling plugin "${manifest.name}" of type "${pSource.type}"...`);
+            this.#plugins[pSource.type] = pSource;
+        }
+
+        return entries;
```
Comment (oleh-momot) [inline ilc/server/pluginManager/PluginManager.js:52]: Does it exactly need to return when this is not used in the constructor?

File: ilc/server/logger/fastify.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    const hyperid = require('hyperid')
+    const instance = hyperid({urlSafe: true, fixedLength: true});
+
+    fastifyConf = {
+        logger: require('./index'),
+        genReqId: instance,
+    }
+}
+
+module.exports = _.omit(_.pick(fastifyConf, ['logger', 'requestIdLogLabel', 'genReqId']), _.isEmpty);
```
Comment (oleh-momot) [inline ilc/server/logger/fastify.js:18]: Where does requestIdLogLabel exist?
Should we add the default one?

Comment (StyleT) [reply ilc/server/logger/fastify.js:18]: See https://www.fastify.io/docs/v2.13.x/Server/#requestidloglabel

**Review by oleh-momot — APPROVED** [review]

**Review by nightnei — APPROVED** [review]

**Pull request merged**